handler/src/main/java/io/netty/handler/ssl/SniHandler.java

```diff
@@ -56,7 +58,7 @@
             InternalLoggerFactory.getInstance(SniHandler.class);
     private static final Selection EMPTY_SELECTION = new Selection(null, null);
 
-    protected final AsyncMapping<String, SslContext> mapping;
+    protected final SslContextMapping mapping;
```

Comment:
My assumption is it is not commonly directly referenced, and since you introduced this change we know you don't depend upon it, or are at least willing to absorb the change.

I'm inclined to allow it, but I have a hunch @normanmaurer may not be so lenient.

good reminder to only expose what is necessary https://github.com/netty/netty/pull/5786#r77206033 ... especially member variables which are very difficult to change after the fact.